mountpoint-s3.yaml

```diff
@@ -14,9 +14,10 @@ environment:
       - ca-certificates-bundle
       - clang-19
       - cmake
-      - fuse3
-      - fuse3-dev
+      - fuse2
+      - fuse2-dev
```

Comment:
fuse2-dev will pull in fuse2, we can omit the first one. 

```bash
/ # apk add fuse2-dev
(1/3) Installing fuse-common (3.16.2-r3)
(2/3) Installing fuse2 (2.9.9-r0)
(3/3) Installing fuse2-dev (2.9.9-r0)
OK: 15 MiB in 18 packages
```
also there's `fuse2-static` in index. I think that's not required in this case.